app/views/inspections/index.html.erb

```diff
@@ -11,14 +16,17 @@
   <tbody>
     <% @inspections.each do |inspection| %>
       <tr>
+        <td><%= inspection.item&.name %></td>
+        <td><%= format_datetime_value(inspection.inspection_date) %></td>
+        <td>
+          <span class="label <%= inspection_label(inspection) %>">
+            <%= inspection.status %>
+          </span>
+        </td>
         <td><%= table_button_link 'Show', inspection %></td>
         <td><%= table_button_link 'Edit', edit_inspection_path(inspection) %></td>
         <td><%= table_button_link 'Destroy', inspection, method: :delete %></td>
```

Comment:
I've added permissions.  I think the helper(s) will need pretty careful design to do this cleanly.